tasks/config/tasks.js

```diff
@@ -87,11 +97,11 @@ module.exports = {
 
     /* Comment Banners */
     header: {
-      year       : (new Date()).getFullYear(),
-      title      : release.title,
-      version    : release.version,
-      repository : release.repository,
-      url        : release.url
+      year       : nullish(config.header.year,(new Date()).getFullYear()),
+      title      : nullish(config.header.title, release.title),
+      version    : nullish(config.header.version, release.version),
```

Comment:
is this compatible with https://github.com/fomantic/Fomantic-UI/blob/1c80027fc4cfaa894f7acad48cba59d64c555816/tasks/config/tasks.js#L65 and other replaces?

Reply:
Should be, those values are for the central banner. The regexes you are reffering to are for the existing headers in each component